components/colorPalette/colors.css

```diff
@@ -1,30 +1,31 @@
 :root {
-    --red-color: red;
-    --online-green: rgb(18, 172, 18);
-    --away-yellow: #fca311;
-    --offline-gray: lightgray;
-    --whitesmoke-color: whitesmoke;
-    --error-red: #EB1400;
-    --warning-yellow: #ff9100;
-    --info-blue: #0077b6;
-    --success-green: #2b9348;
-    --primary-blue: #0066ff;
-    --primary-hover-blue: #0466c8;
-    --primary-blue-disabled-color: rgba(0, 102, 255, 0.39);
-    --whitesmoke-color: whitesmoke;
-    --dark-theme-bg-color: #121212;
-    --secondary-dark-blue: #1d2d44;
-    --secondary-dark-blue-hover: #33415c;
-    --secondary-dark-blue-disabled: rgb(29, 45, 68, 0.39);
-    --success-green: #268541;
-    --white-color: white;
-    --lightish-gray: rgb(66, 62, 62);
+  --red-color: red;
+  --online-green: rgb(18, 172, 18);
```

Comment:
As a good practice in CSS, you can use 1 type of color code. You can change these RGB values to Hex codes like the other ones.

Reply:
Hey Gaurav. Thank you for the suggetsion. Will change rgb() to HEX code wherever alpha value is not used